raiden/ui/app.py

```diff
@@ -341,17 +341,19 @@ def run_app(
 
     config['database_path'] = os.path.join(database_path, f'v{RAIDEN_DB_VERSION}_log.db')
 
+    log_file_path = os.path.join(database_path, 'debug.log')
+    log_file_path = kwargs.get('log_file') if kwargs.get('log_file') else log_file_path
```

Comment:
This line could become:
`log_file_path = kwargs.get('log_file', log_file_path)`